.github/workflows/dynamiclinks.yml

```diff
@@ -36,3 +36,22 @@ jobs:
       run: scripts/setup_bundler.sh
     - name: PodLibLint Storage Cron
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseDynamicLinks.podspec --platforms=ios ${{ matrix.flags }}
+
+  quickstart:
```

Comment:
The test is failing. There may be missing steps in the setup. It should be sorted out or removed for a subsequent PR.

Reply:
remove it for now will add it back later.